src/Paket.Core/VSIntegration.fs

```diff
@@ -3,39 +3,46 @@
 open System.IO
 open Logging
 open System
+open Rop
+open Domain
 
-let InitAutoRestore(dependenciesFileName) =
-    let root =
-        if dependenciesFileName = Constants.DependenciesFileName then
-            "."
-        else
-            Path.GetDirectoryName dependenciesFileName
 
-    CreateDir(Path.Combine(root,".paket"))
+let getLatestVersionFromJson (data : string) =
+    try 
+        let start = data.IndexOf("tag_name") + 11
+        let end' = data.IndexOf("\"", start)
+        (data.Substring(start, end' - start)) |> SemVer.Parse |> succeed
+    with _ ->
+        fail ReleasesJsonParseError
+
+let InitAutoRestore environment =
+    let exeDir = Path.Combine(environment.RootDirectory.FullName, ".paket")
+    CreateDir(exeDir)
```

Comment:
Btw. I don't like to have this one explicitly called, for me it should be part of `download` function.
No one needs this directory to be present besides the download, as it would fail.